### PR TITLE
Register additional endpoints for nuts-patient-transfer (Nedap)

### PR DESCRIPTION
**Pull request by jelmerterwal**

### CODE REVIEW

File: events/20200427130105931-RegisterEndpointEvent.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "type": "RegisterEndpointEvent",
+  "issuedAt": "2020-04-27T13:01:05.931322217Z",
+  "payload": {
+    "organization": "urn:oid:2.16.840.1.113883.2.4.6.1:48000010",
+    "URL": "https://api-development.ons.io/t/fhir/",
```
Comment (jelmerterwal) [inline events/20200427130105931-RegisterEndpointEvent.json]:
```suggestion
    "URL": "https://api-staging.ons.io/t/fhir/",
```

File: events/20200427130111702-RegisterEndpointEvent.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "type": "RegisterEndpointEvent",
+  "issuedAt": "2020-04-27T13:01:11.702780774Z",
+  "payload": {
+    "organization": "urn:oid:2.16.840.1.113883.2.4.6.1:48000020",
+    "URL": "https://api-development.ons.io/t/fhir/",
```
Comment (jelmerterwal) [inline events/20200427130111702-RegisterEndpointEvent.json]:
```suggestion
    "URL": "https://api-staging.ons.io/t/fhir/",
```

**Comment by woutslakhorst** [comment]
from our conversation on Slack, I understood most of these are intended for the test network?

**Comment by jelmerterwal** [comment]
correct, so these will be altered a bit :)

File: 20200427115904807-RegisterEndpointEvent.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (reinkrul) [inline 20200427115904807-RegisterEndpointEvent.json]: deze file moet volgens mij in ./events staan?

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: events/20200423121631337-RegisterEndpointEvent.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "type": "RegisterEndpointEvent",
+  "issuedAt": "2020-04-23T12:16:31.337857222Z",
+  "payload": {
+    "organization": "urn:oid:2.16.840.1.113883.2.4.6.1:48000001",
+    "URL": "tcp://nuts-development.ons.io",
+    "endpointType": "urn:nuts:endpoint:consent",
+    "identifier": "urn:ietf:rfc:1779:O=Nedap,C=NL,L=Groenlo,CN=nedap_nuts_cordapp_development",
+    "status": "active"
+  }
```
Comment (reinkrul) [inline events/20200423121631337-RegisterEndpointEvent.json:10]: This file is malformed; it's missing the end accolade ( } ). By the way, you can test this by starting the **nuts** binary with this repository as registry datadir. I use something like this:

`$ NUTS_REGISTRY_DATADIR=<path to registry directory> NUTS_IDENTITY=urn:oid:1.3.6.1.4.1.54851.4:08013836 NUTS_AUTH_ACTINGPARTYCN=foo NUTS_AUTH_PUBLICURL=bar ./nuts`

**Review by reinkrul — CHANGES_REQUESTED** [review]

File: events/20200423121631337-RegisterEndpointEvent.json (hunk 4 above)
Comment (jelmerterwal) [reply events/20200423121631337-RegisterEndpointEvent.json:10]: the generated files probably lack an ending newline character, as I copy + pasted it here :)

**Review by reinkrul — APPROVED** [review]
@woutslakhorst files look and load fine. Do you have any additional commnets?

**Comment by woutslakhorst** [comment]
may be merged if still needed. I believe endpoints are also added for Test for the same use-case

**Pull request merged**